machine_learning_hep/steer_analysis.py

```diff
@@ -124,6 +124,7 @@ def do_entire_analysis(data_config: dict, data_param: dict, data_param_overwrite
     dohistomassmc = data_config["analysis"]["mc"]["histomass"]
     dohistomassdata = data_config["analysis"]["data"]["histomass"]
     doefficiency = data_config["analysis"]["mc"]["efficiency"]
+    efficiency_resp = data_config["analysis"]["mc"]["efficiency_resp"]
```

Comment:
This will fail if the key is not present in the database, which will be the case for non-jet analyses.

Reply:
How has this been addresses?